tests/rest_api/test_0001_users_api.py

```diff
@@ -12,12 +12,14 @@
 
 def test_non_admin_cannot_see_others():
     for username in ['dummy1', 'worker1', 'user1', 'business1']:
-        response = requests.get(config.get_api_url('users'), auth=(username, config.USER_PASS))
+        response = requests.get(config.get_api_url('users'), auth=(username, config.USER_PASS),
```

Comment:
@Marishka17 , tests should test all versions. Thus if we have a default version, right now it will be enough. Let's discuss. Really want to find the right approach.